Modelica/Mechanics/MultiBody/Examples/Loops/Utilities.mo

```diff
@@ -798,99 +798,122 @@ solved analytically.
 </html>"));
   end EngineV6_analytic;
 
-  partial model Engine1bBase "Model of one cylinder engine with gas force"
+  partial block Engine1aBase
+    "Common model of one cylinder engine without gas force."
```

Comment:
Not necessary to say "without gas force" here. This is simply a base model of ANY one cylinder engine. So actually, the proper description shall be: "Base model for one cylinder engine".